src/xx-info

```diff
@@ -337,6 +340,14 @@ case "$TARGETARCH" in
     XX_RHEL_ARCH="mips64el"
     XX_TRIPLE="mips64el${vendor}-linux-${XX_LIBC}abi64"
     ;;
+  "loong64")
+    XX_MARCH="loong64"
+    XX_DEBIAN_ARCH="loong64"
+    XX_ALPINE_ARCH="loongarch64"
+    XX_RHEL_ARCH="loong64"
+    XX_TRIPLE="loong64${vendor}-linux-${XX_LIBC}abi"
```

Comment:
According to https://wiki.debian.org/Ports/loong64, at least the debian triple is `loongarch64-linux-gnu`

Reply:
thanks！